Source/Extensions/Int+DecimalSequence.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  Int+DecimalSequence.swift
+//  TickerCounter
+//
+//  Created by Kevin Miller  on 3/18/19.
+//
+
+import Foundation
+
+extension Int {
+    
+    func ascendingDecimalSequenceTo(_ end: Int) -> [Int] {
+        guard
+            self <= 9,
+            end <= 9,
+            self >= 0,
+            end >= 0 else { return [] }
+        let decimalDigits = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
```

Comment:
food for thought - it seems like `0` thru `9` are significant digits and they're used over and over; in my refactoring, i usually see that as a sign that they may need to be named in a certain way that documents their significance